framework/src/modules/governance/events/proposal_creation_failed.ts

```diff
@@ -1,17 +0,0 @@
-/*
- * Copyright © 2023 Lisk Foundation
- *
- * See the LICENSE file at the top-level directory of this distribution
- * for licensing information.
- *
- * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
- * no part of this software, including this file, may be copied, modified,
- * propagated, or distributed except according to the terms contained in the
- * LICENSE file.
- *
- * Removal or modification of this copyright notice is prohibited.
- */
-
-import { BaseEvent } from '../../base_event';
-
-export class ProposalCreationFailedEvent extends BaseEvent<void> {}
```

Comment:
why was this deleted?

Reply:
As I wrote in the PR description, this event is being removed in the [upcoming PR](https://github.com/LiskHQ/lips/pull/460).

Based on my suggestion to move `hasEnded` check from `execute` to `verify` part of `CreateProposal` command, this event is no longer necessary.